cms/envs/common.py

```diff
@@ -2725,3 +2725,7 @@
 
 DISCUSSIONS_INCONTEXT_FEEDBACK_URL = ''
 DISCUSSIONS_INCONTEXT_LEARNMORE_URL = ''
+
+#### django-simple-history##
```

Comment:
https://github.com/cloudtruth/django-simple-history/blob/b45c6cc8442038b03be6ef1b3619120cecdd61f2/docs/historical_model.rst#indexed-history_date

Don't need this for this PR. Owning team can decide this in future.